.gitignore

```diff
@@ -6,6 +6,7 @@
 # Folders
 _obj
 _test
+tests/cilium-files
```

Comment:
@ianvernon I've spoked with @ashwinp it seems there are some tests files laying around. Can we remove them?

Reply:
Anything in `tests/cilium-files` is created by the tests and can be removed / ignored.